README.md

```diff
@@ -167,6 +168,37 @@ First, [fork](https://guides.github.com/activities/forking/) this repository, in
 When you make a PR to edit an existing package, `dt-bot` should @-mention previous authors.
 If it doesn't, you can do so yourself in the comment associated with the PR.
 
+#### Editing tests on an existing package
+
+Any `*.ts` files in the package are considered as "test" files.
+These files are used to validate the API exported from the `*.d.ts` files which are shipped as `@types/yourmodule`.
+
+Changes to the `*.d.ts` files should include a corresponding `*.ts` file change which shows the API being used, so that someone doesn't accidentally break code you depend on.
+
+For example, this change to a function in a `.d.ts` file adding an option
+
+`index.d.ts`:
+
+```diff
+- export function twoslash(body: string): string
++ export function twoslash(body: string, config?: { version: string }): string
+```
+
+`index-tests.ts`:
+```diff
+import {twoslash} from "./"
+
+// ExpectType string
+const result = twoslash("//")
+
++ // Handle options param
++ const resultWithOptions = twoslash("//", { version: "3.7" })
++ // When the param is incorrect
++ // $ExpectError
++ const resultWithOptions = twoslash("//", {  })
+```
+
+You can validate your changes with `npm run lint package-name` in the root of the repo.
```

Comment:
This will not run tests, sadly. But excellent starting point!
Readme could promote running `npm test` (simple). This will run tests only againsts current changes (e.g. againts files added to the `types/{module}`.
As the DT repository changes frequently, in order to succesfully run local tests, one needs to:
- update from remote before starting new work (so we start with fresh, udpated branch)
- [optionally] rebase
That is crucial for modules, that reference or are referenced by popular packages, like `@types/react`.